ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -746,9 +746,6 @@ export default class ConfirmTransactionBase extends Component {
 
         sendTransaction(txData)
           .then(() => {
-            if (!this._isMounted) {
```

Comment:
In MM, the sendTransaction resolve promise lacks of logic. In MMI, additional actions are required. Hence, we relocated the `isMounted` exactly where we updated the state.